e2e/vfolder.test.ts

```diff
@@ -8,14 +9,72 @@ import {
   restoreVFolderAndVerify,
   shareVFolderAndVerify,
   userInfo,
-} from './test-util';
-import { test } from '@playwright/test';
+} from './utils/test-util';
+import { test, expect } from '@playwright/test';
 
 test.describe('VFolder ', () => {
   test.beforeEach(async ({ page }) => {
     await loginAsUser(page);
   });
   const folderName = 'e2e-test-folder-user-creation' + new Date().getTime();
+  test.describe('vFolder Creation', () => {
+    test.beforeEach(async ({ page }) => {
+      await page.getByRole('link', { name: 'Data' }).click();
+      await page.getByRole('button', { name: 'Create Folder' }).nth(1).click();
+    });
+    test.afterEach(async ({ page }) => {
+      await moveToTrashAndVerify(page, folderName);
+      await deleteForeverAndVerifyFromTrash(page, folderName);
+    });
+    test('User can create a vFolder by selecting a specific location', async ({
+      page,
+    }) => {
+      const folderCreationModal = new FolderCreationModal(page);
+      await folderCreationModal.modalToBeVisible();
+      await folderCreationModal.fillFolderName(folderName);
+      await folderCreationModal.fillLocationSelector('local');
+      await folderCreationModal.selectLocationOptionByText('local');
+      await folderCreationModal.clickCreateButton();
+    });
+    test('User can create default vFolder', async ({ page }) => {
+      const folderCreationModal = new FolderCreationModal(page);
+      await folderCreationModal.modalToBeVisible();
+      await folderCreationModal.fillFolderName(folderName);
+      await folderCreationModal.clickCreateButton();
+    });
+    test('User can create Model vFolder', async ({ page }) => {
+      const folderCreationModal = new FolderCreationModal(page);
+      await folderCreationModal.modalToBeVisible();
+      await folderCreationModal.fillFolderName(folderName);
+      await folderCreationModal.selectModelUsageMode();
+      await folderCreationModal.clickCreateButton();
+    });
+    test('User can create cloneable Model vFolder', async ({ page }) => {
+      const folderCreationModal = new FolderCreationModal(page);
+      await folderCreationModal.modalToBeVisible();
+      await folderCreationModal.fillFolderName(folderName);
+      await folderCreationModal.selectModelUsageMode();
+      await folderCreationModal.clickCloneableSwitchButton();
```

Comment:
Instead of add `clickCloneableSwitchButton` function for clicking, it's enought to use like to.

```suggestion
      await (await folderCreationModal.getCloneableSwitchButton()).click();
```